src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -105,4 +115,19 @@ const StyledIcon = styled(Icon)`
   }
 `;
 
+const Notice = styled.div`
+  margin-left: 184px;
+  display: flex;
+`;
+
+const AlertIcon = styled(Icon)`
+  color: ${({ theme }) => theme.colors.dark.functional.attention};
+`;
```

Comment:
You don't need to use styled components. Just use the Icon component directly and it has a color prop you can pass the color into.